app/views/confirmables/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
you've converted all these files to be HAML files. So we don't need these .erb files anymore. When you ran the conversion script it would have asked you if you'd like to remove the erb files. That would have been easier to do - but otherwise all .erb files can be manually removed also from the codebase.

Reply:
oh...i just saw these new files now. i had made all erbs haml but when i removed the comments of the confirmable and trackable in the user model and ran the devise generate views for confirmable, by default it made some new erb files instead of haml that i wasn't aware of. will convert them